std_msgs/String class_
std_msgs/int32 x
std_msgs/int32 y
std_msgs/int32 h
std_msgs/int32 w